Trying to clean-up conda-smithy results

sodre opened the pull request:
@conda-forge-admin, please rerender

<!--
Thank you for pull request.
Below are a few things we ask you kindly to self-check before getting a review. Remove checks that are not relevant.
-->
Checklist
* [x] Used a fork of the feedstock to propose changes
* [x] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/conda_smithy.html#how-to-re-render ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

<!--
Please note any issues this fixes using [closing keywords]( https://help.github.com/articles/closing-issues-using-keywords/ ):
-->

<!--
Please add any other relevant info below:
-->

sodre committed:
Remove |lower and .lower

sodre committed:
Fix win compiler packages for windows

sodre committed:
MNT: Re-rendered with conda-smithy 3.2.2 and pinning 2018.12.18

sodre committed:
Use target_platform

sodre committed:
Bump up build number

conda-forge-linter commented:
Hi! This is the friendly automated conda-forge-linting service.

I wanted to let you know that I linted all conda-recipes in your PR (```recipe```) and found some lint.

Here's what I've got...


For **recipe**:

 * Failed to even lint the recipe, probably because of a conda-smithy bug :cry:. This likely indicates a problem in your `meta.yaml`, though. To get a traceback to help figure out what's going on, install conda-smithy and run `conda smithy recipe-lint .` from the recipe directory.

conda-forge-linter commented:
Hi! This is the friendly automated conda-forge-webservice.

I tried to re-render for you, but it looks like there was nothing to do.

sodre committed:
MNT: Re-rendered with conda-build 3.16.3, conda-smithy 3.2.2.post.dev11, and conda-forge-pinning 2018.12.18

conda-forge-linter commented:
Hi! This is the friendly automated conda-forge-linting service.

I wanted to let you know that I linted all conda-recipes in your PR (```recipe```) and found some lint.

Here's what I've got...


For **recipe**:

 * Failed to even lint the recipe, probably because of a conda-smithy bug :cry:. This likely indicates a problem in your `meta.yaml`, though. To get a traceback to help figure out what's going on, install conda-smithy and run `conda smithy recipe-lint .` from the recipe directory.

sodre committed:
Enable go tests

sodre committed:
Disable tests

  - On linux the following fails:
```
--- FAIL: TestDialTimeout (0.02s)
	dial_test.go:100: all connections connected; expected some to time out
FAIL
FAIL	net	2.246s
```

  - On OSX the following fails:
```
--- FAIL: TestGetfsstat (0.00s)
	syscall_bsd_test.go:31: an empty Statfs_t struct was returned
FAIL
FAIL	syscall	0.016s
```

sodre committed:
Remove C++ and Fortran compilers

sodre committed:
MNT: Re-rendered with conda-build 3.16.3, conda-smithy 3.2.2.post.dev11, and conda-forge-pinning 2018.12.18
